test/data/sidechain_testdata.js

```diff
@@ -386,4 +386,74 @@ module.exports = [
beca3da39a346705065b8c0751e257a1c310a6ff013a4034b3880f000399450eb40001640000013f04f605000000002b04ec06755daa1a12b32c995d027b3e9cbfcd0aeb576d38e8cc7123aca9705b21dd2848350f5a2988bba858f711c3fc1df6f11efaf495d26ff381fa197d74b312000120ceaf4c7c219fba7ac83eead460e3d20da943e43cfc03e8bb9da9a291d3525129fdcd01026e4c0000000000000400000000000000724c000000000000625f0100000000000c00000000000000015b66bac4fa639fd610a77dde0b8a5b8aee1a9045e4d8edc253b48dc1973eaa17800001f2ba59ab3f8c17b521cf0d7f50503184534bebd5ee4346ef6ed6ec20316b770c800001178c98cd6b079518f4fac4db75f30b529b6a9a758d0ff3a78828f730efb9553e000001b054544e2281a38c7726a8b4bb06374fb7bc188c747957f99dd73c01ec6fe4368000010cbd5079b83ab23f71df62758916b5fe43599b7799bf525c818798906ac5363200000187f68fcea13a27ff26ea9ac738c8cbb7e3b66a8ae2da9b08cb76085bebe74a1300000100aaaab552e6d0e08186b700d6032908c8150b47005f81173a57032a1603eb33800001236652c568f791fe248c4613cdf6fbc7b746285d7f475ff3c564b06de5f7653d800001786ff30c68d2261060030f9aa1b2263572008b6c0ed9e73f623d3f4b8fc42a378000010cf4f3e6bf50a7ae2b995fd741b700e4792833ea9d748918d6a45db01d05433080000120ca3381b5081425c1a5611f33b9cdfb2a48dfe0a00fca3b4430f8e1f238d3348000015b576e85c64f1ee983c594045d8448aaef68900e301054936c5477e86dfaf512000001fd590302a7b90300000000000400000000000000abb90300000000005a450600000000000c0000000000000002d1ad1b36532a682e18b77adfb4a000fce394ae5bcff18116b9e49dbe79c6062f806f5f8c84c3fcba4402c1845a764341a6b2a614251c37d0384bdfa30b463c941400000244e5ce84fa4026cdad41016676e5769b0f0efd30265ea4278b187ce10ddde5360098470c208fdba4676901963be38731d8a14591e4845e6ef57bfa89e845015d0c800002772174c8a7f9d2667444c523a3d2d4581a449c87b46c2a52a3b5806bd993ce07809e4de5697418aac9be0715d1a141b21913edbce955e0320ca367f0e6e21475130000024fc6b754db64b057c60daea4b6a20926f1745b84a6431a25490d4ea161ca6405806399fc98c20fc73feb46728ab1dca306d002ee6809e8a55af5f9f0d26b08153f000002b663eccd1750e1cc78e0be484e70f5cfb465ab10f4a4ba3b7c23922d08f2213e808572917b014d67213cb5cc2fa575a356870b6727af0b11de270a8179df55af2a000002ff584a87d4dca3f1de55301fdc288bd9520a56023c67825fe33d542d6d73d03580d4efd6dbc1482222f38aa8a3606b4ef6f42376f64f22863a43b9ae076dd9942a000002fd7dfc42456761b77a941fe287ea11b8940187a04e8c6cc2c52cefd567591321806fa775b12a2d649b301a8cb9f4290eda166857c7fda5b4d2021f1615ea47ee2f8000024dccd2df509bc92cec0b9784861f1b32403532319b731173d142550b4305be1d8070b4e5e355f72666917aa41d5f77670a0a6b8f1d8961b7f18ce9414a63195010800002d119df8e835abe10e6fb40295631d0207590898538a1d6b7722de8276bef4b0d00ce2cd406733d46e7fcafbc5e93f8d87b6ee387da3a609115f38d13e7c002e63b0000026030a79f592f0deea3ab16a17ee39041e0482942295f579d051438e8823463038019128570a317f32d83f6e603739f8d1eedcc19a5409127f87c13ac6b18ac972b800002e694fb8713ed7625a964791ae15671a53d2ed7c328352b3e888b0e2d8240a103800430b8aaa7dbf5411aa88b00dd15377db1dbb6d721fe49b1829cd24bca96d425800002e9656a50b86f3001fdf7c521d24fa19360e3189ca8b939fc901896b2289e4728802d8030df1724cde8645e6d1df9a51258234386b770ff243b9cfb952eda1d5e17000002ffff000000000000000000000000000000000000000000000000",
```

Comment:
We can delete the other test data for `"Sidechain creation TX"` bc the data is for the wrong type of transaction